src/main/kotlin/com/zepben/evolve/cim/iec61968/metering/ControlledAppliance.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61968.metering
+
+import kotlin.reflect.full.declaredMemberProperties
+
+/**
+ * Appliance controlled with a PAN device control.
+ *
+ * @property isElectricVehicle True if the appliance is an electric vehicle.
+ * @property isExteriorLighting True if the appliance is exterior lighting.
+ * @property isGenerationSystem True if the appliance is a generation system.
+ * @property isHvacCompressorOrFurnace True if the appliance is HVAC compressor or furnace.
+ * @property isInteriorLighting True if the appliance is interior lighting.
+ * @property isIrrigationPump True if the appliance is an irrigation pump.
+ * @property isManagedCommercialIndustrialLoad True if the appliance is managed commercial or industrial load.
+ * @property isPoolPumpSpaJacuzzi True if the appliance is a pool, pump, spa or jacuzzi.
+ * @property isSimpleMiscLoad True if the appliance is a simple miscellaneous load.
+ * @property isSmartAppliance True if the appliance is a smart appliance.
+ * @property isStripAndBaseboardHeater True if the appliance is a strip or baseboard heater.
+ * @property isWaterHeater True if the appliance is a water heater.
+ */
+data class ControlledAppliance(
```

Comment:
There's a few other public data classes with defaults and no `@JvmOverloads` but just fix this one for now:
```suggestion
data class ControlledAppliance @JvmOverloads constructor(
```

Reply:
added